server/routes/replays.js

```diff
@@ -123,6 +123,7 @@ router.get('/:replayId/export', async (req, res, next) => {
  */
 // eslint-disable-next-line consistent-return
 router.post('/', (req, res, next) => {
+    req.query.mapName = decodeURI(req.query.mapName);
```

Comment:
Small (mostly aesthetic) change: Can we use `decodeURIComponent` instead?
The logic is very similar, but `decodeURI` is meant to be used on a complete URI - while we're only operating on a component of it.

Also I don't love reassigning the request property, but I don't care that much if you want to keep it for simplicity.